docs/source/usage.rst

```diff
@@ -20,18 +20,6 @@ You can import the package using the following line of code:
 
     import indieweb_utils
 
-Find a Post Type
-------------------------
-
-To find the post type associated with a web page, you can use the `get_post_type` function.
-
-The `get_post_type` function function uses the following syntax:
-
-.. autofunction:: indieweb_utils.get_post_type
-
-This function returns a single string with the post type of the specified web page.
-
-See the `Post Type Discovery specification <https://indieweb.org/post-type-discovery>`_ for a full list of post types.
 
 Custom Properties
```

Comment:
Not sure where custom properties should go?

Reply:
I have moved them under the get_post_type definition.